packages/theme-addresses/README.md

```diff
@@ -0,0 +1,92 @@
+# @shopify/theme-addresses
+
+## Getting Started
```

Comment:
Lets include a brief description of what the library does and where it can be used. See https://github.com/Shopify/theme-scripts/tree/master/packages/theme-cart

Need to do this with some other libs in this repo too since they are missing.